### ISSUE TITLE
The first parameter of operator new must be size_t

### ISSUE THREAD

**Issue by syby119** [DrTimothyAldenDavis/SuiteSparse#533]
In the project GPUQREngine, when I compile it with MSVC 2022, the compiler tells me that the following operator new function with error declaration:
```
void* BucketList<Int>::operator new(long unsigned int, BucketList <Int>* p);
void* LLBundle<Int>::operator new(long unsigned int, LLBundle <Int>* p);
void* Font<Int>::operator new(long unsigned int reqMem, Front* ptr);
void* Scheduler<Int>::operator new(long unsigned int, Scheduler <Int>* p);
```

I think the main problem here is that the type **long unsigned int** equals to **uint32_t** on my Windows 11, whereas **size_t** equals to **uint64_t**.

To fix the problem, simply replace **long unsigned int** to **size_t**.

Consider the following code as the simplest example. On a platform with **long unsigned int** equals to **uint32_t**, the code won't be compiled.
```
template <typename T>
class A {
    void *operator new(unsigned long int size, int* p){ return p; }
};

int main() {
    A<int> a;
}
```

**Comment by DrTimothyAldenDavis** [comment]
This is now fixed in SuiteSparse 7.4.0 -- thanks for the update.

**Comment by DrTimothyAldenDavis** [comment]
(SuiteSparse 7.4.0 is in the default dev branch but not yet made into a 7.4.0.beta1 release; I'll do that soon).
